### PR TITLE
Update home page layout and hub icon classes

### PR DESCRIPTION
**Pull request by jbalboni**
## Description

I started making homepage updates, but rather than create another place where we put icon classes in markdown frontmatter, I went ahead and make the changes to create hub icon classes.

This also updates the disability icon, per this ticket: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/14484

## Testing done
Viewed homepage locally

## Screenshots

localhost_3001_ 6

## Acceptance criteria
- [x] Homepage still looks the same
- [x] Hub icons are showing up in the correct places
- [x] Other icons are not broken

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/site-wide/sass/iconography.scss [hunk 1]
```diff
@@ -8,202 +8,153 @@ h1, h3, h4 {
...
+
+@include make-hub(
+  $hub-name: "records",
+  $icon: $fa-var-id-card,
+  $color: $violet-warm-60,
+  $top-small: 7px,
+  $left-small: 6px,
+  $top-large: 10.25px,
+  $left-large: 9.5px
+);
```
Comment (ncksllvn) [inline src/platform/site-wide/sass/iconography.scss]: 🔥 🔥 🔥 🔥 🔥

File: va-gov/pages/index.md [hunk 2]
```diff
@@ -5,12 +5,118 @@ title: Home
...
+    title: Health Care Access and Quality
+    href: https://www.accesstocare.va.gov/
+    description: Get wait times and compare quality and patient-satisfaction data for VA health facilities.
+  - img: /img/homepage/home-3.jpg
+    alt: Close-up of an American flag patch on the sleeve of a U.S. military combat uniform.
+    title: VA Rapid Appeals
+    href: https://www.blogs.va.gov/VAntage/48141/vas-rapid-appeals-modernization-program-ramp-now-open-appeals/
+    description: Find out how to get a faster decision on a pending disability compensation appeal through the VA Rapid Appeals Modernization Program (RAMP).
+
+
```
Comment (ncksllvn) [inline va-gov/pages/index.md]: 🔥 🔥 🔥 🔥 🔥 🔥

**Review by ncksllvn — APPROVED** [review]
Awesome work!! Looks great. Love seeing the hub styles finally consolidate into something that we can work with more easily.

**Review by amycesal — APPROVED** [review]

File: va-gov/pages/index.md (hunk 2 above)
Comment (bethpotts) [reply va-gov/pages/index.md]: This looks great @jbalboni --no edits from me.

**Pull request merged**